debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresChangeRecordEmitter.java

```diff
@@ -234,8 +236,8 @@ private void refreshTableFromDatabase(TableId tableId) {
         }
     }
 
-    static Optional<DataCollectionSchema> updateSchema(PostgresPartition partition, TableId tableId, ChangeRecordEmitter<PostgresPartition> changeRecordEmitter) {
-        return ((PostgresChangeRecordEmitter) changeRecordEmitter).newTable(tableId);
+    static Optional<DataCollectionSchema> updateSchema(Partition partition, DataCollectionId tableId, ChangeRecordEmitter<PostgresPartition> changeRecordEmitter) {
```

Comment:
are these changes on postgres connector related?

Reply:
I have the same issue D

Reply:
@ggaborg I've solved it via even smaller change now :-)

Reply:
This is how I've solved it too. But do you want to keep this change in this DBZ PR?

Reply:
Yeah, it is completely harmaless, non-functional change.